back/spec/services/idea_custom_fields_service_spec.rb

```diff
@@ -86,17 +86,18 @@
     describe 'enabled_public_fields' do
       it 'excludes disabled & answer_visible_to: admins fields' do
         output = service.enabled_public_fields
-        expect(output.map(&:code)).to eq %w[
-          title_page
-          title_multiloc
-          body_page
-          body_multiloc
-          uploads_page
-          idea_images_attributes
-          idea_files_attributes
-          details_page
-          topic_ids
-          location_description
+        expect(output.map(&:code)).to eq [
+          'title_page',
+          'title_multiloc',
+          'body_page',
+          'body_multiloc',
+          'uploads_page',
+          'idea_images_attributes',
+          'idea_files_attributes',
+          'details_page',
+          'topic_ids',
+          'location_description',
+          nil
```

Comment:
The end page used to not be publicly visible, whereas other pages were publicly visible. I didn't see any reason for this and tested that making the end page publicly visible wouldn't cause issues. So I changed this, so all pages are always publicly visible.

I suspect this behaviour was unintentional no visibility was specified in the default form of ideation.rb, whereas for the other fields visibility gets set to public.